WooCommerce/Classes/ViewRelated/Coupons/Add and Edit Coupons/AddEditCoupon.swift

```diff
@@ -64,12 +103,13 @@ struct AddEditCoupon: View {
                             .padding(.bottom, Constants.verticalSpacing)
 
                             Button {
-                                //TODO: handle action
+                                showingEditDescription = true
                             } label: {
                                 HStack {
-                                    Image(uiImage: .plusImage)
+                                    Image(uiImage: viewModel.editDescriptionIcon)
+                                        .colorMultiply(Color(.text))
```

Comment:
❓ Why should we use `colorMultiply` here instead of a simple `foregroundColor`?

Reply:
The reason is that some images, like `pencilImage`, are modified with the tint color during the creation, so the `foregroundColor` modifier wouldn't work.